StreamChatClient_v3/WebSocketClient/Events/Event.swift

```diff
@@ -33,3 +33,8 @@ protocol EventWithChannelId: Event {
 protocol EventWithMessagePayload: EventWithUserPayload, EventWithChannelId {
     var messageId: MessageId { get }
 }
+
+/// A protocol for member `Event`s.
+protocol EventWithMemberPayload: EventWithPayload {
+    var userId: UserId { get }
```

Comment:
this should probably be `memberId`?

Reply:
Member doesn't have `memberId`. In `MemberDTO` you do this: `channeldId.rawValue + userId`